modules/react-native-chat/options.js

```diff
@@ -0,0 +1,72 @@
+import { Platform, StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  textInput: {
+    paddingVertical: 4,
+    paddingHorizontal: 4
+  },
+  section: {
+    paddingHorizontal: 8,
+    paddingVertical: 8,
+    marginVertical: 8,
+    marginHorizontal: 8,
+    backgroundColor: 'white',
+    borderRadius: 6
+  }
+});
+export const NavigationStyle = StyleSheet.create({
+  title: {
+    fontSize: 16
+  },
+  headerRight: {
+    marginTop: 5,
+    paddingRight: 16
+  }
+});
+export const ListViewStyle = StyleSheet.create({
+  container: {
+    paddingVertical: 8,
+    paddingHorizontal: 8
+  },
+  title: {
+    color: '#333',
+    fontSize: 16,
+    fontWeight: '600',
+    paddingBottom: 4
+  },
+  subtitle: {
+    color: '#3d3d3d',
+    fontSize: 16,
+    paddingBottom: 4,
+    paddingTop: 6
+  },
+  circle: {
+    backgroundColor: '#faa',
+    borderRadius: 50,
+    width: 50,
+    height: 50,
+    marginRight: 8
+  },
+  separator: {
+    borderBottomWidth: 0.3,
+    borderBottomColor: 'lightgrey',
+    width: '100%',
+    paddingBottom: 12,
+  }
+});
+
+
+const PUBNUB_PUB = "pub-c-09dc7d68-298a-4dd5-9623-7cc67ddaa937"
+const PUBNUB_SUB = "sub-c-253106be-97d7-11ec-b249-a68c05a281ab"
```

Comment:
Is this a private key? This should be replaced by placeholders if that is the case

Reply:
Yes. It was a private key.
I have replaced by placeholders